specification/inference/post_eis/PostEisRequest.ts

```diff
@@ -0,0 +1,58 @@
+/*
+ * Licensed to Elasticsearch B.V. under one or more contributor
+ * license agreements. See the NOTICE file distributed with
+ * this work for additional information regarding copyright
+ * ownership. Elasticsearch B.V. licenses this file to you under
+ * the Apache License, Version 2.0 (the "License"); you may
+ * not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *    http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+
+import type { Request as RequestChatCompletion } from '../chat_completion_unified/UnifiedRequest'
+import { Id } from '@_types/common'
+
+
+export type OmittedChatCompletion = Omit<RequestChatCompletion, 'urls' | 'path_parts'>;
```

Comment:
@swallez @pquentin we're trying to override the `urls` and `path_parts`, we're getting errors when trying to run `make generate`. Is this a valid solution?

Technically we don't really have to do this, I'm just curious if it's possible. `UnifiedRequest` will likely be identical to the `Request` interface we're defining in this file. We are trying to rename the `inference_id` path part to `eis_inference_id` just to make the documentation a little clearer.

Reply:
An option that looks like it's working is:

```
import type { Request as RequestChatCompletion } from '../chat_completion_unified/UnifiedRequest'

export interface Request extends RequestChatCompletion {}
```